ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/base-template/policies/.content.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:lastModified="{Date}2024-05-22T14:10:52.198-05:00"
+        cq:lastModifiedBy="admin"
+        cq:policy="aem-bootcamp/components/page/policy"
+        jcr:primaryType="nt:unstructured"
+        sling:resourceType="wcm/core/components/policies/mappings">
+        <root
+            cq:policy="aem-bootcamp/components/container/policy_1574694950110"
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="wcm/core/components/policies/mapping">
+            <container
+                cq:policy="aem-bootcamp/components/container/policy_1574695586800"
```

Comment:
Not sure how many policies you should have here? Please check if you need 3 instead of 2.